src/node.cc

```diff
@@ -2379,6 +2415,20 @@ void OnMessage(Local<Message> message, Local<Value> error) {
 }
 
 
+void ClearFatalExceptionHandlers(Isolate* isolate) {
```

Comment:
heh. no reason at all. my brain just defaulted to using public API style parameters.